src/site/content/en/reliable/measuring-offline-usage/index.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: Measuring offline usage
+authors:
+  - giesau
+  - mschierle
+date: 2020-10-22
+tags:
```

Comment:
```suggestion
date: 2020-10-22
description: >
  How to track offline usage so that you can make a case for why your site needs
  a better offline mode.
tags:
```
Forgot to add a description